static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -285,3 +289,43 @@ export function getCompoundDiseaseContraindication(
   processedData.sort((a, b) => (a.drugSource > b.drugSource ? 1 : -1));
   return processedData;
 }
+
+/**
+ * Fetches the common name of the disease of interest
+ * @param data
+ * @returns - string with disease common name
+ */
+export function getDiseaseCommonName(data: GraphNodes): string {
+  let commonName = null;
+  let diseaseDCID = null;
```

Comment:
also here, don't need to define diseaseDCID as null, can just define the variable in line 308

Reply:
Correct! Fixed that!